cohorts/cohort.py

```diff
@@ -1088,9 +1099,11 @@ def get_filtered_isovar_epitopes(self, epitopes, ic50_cutoff):
         return EpitopeCollection(mutant_binding_predictions)
 
     def load_single_patient_isovar(self, patient, variants, epitope_lengths):
-        # TODO: different epitope lengths, and other parameters, should result in
-        # different caches
-        isovar_cached_file_name = "%s-isovar.csv" % self.merge_type
+        # different cache depending on epitope length & variant set
+        variant_hash = make_hash(frozenset(variants))
```

Comment:
Nice! Are we confident that this hashes to the same value for the same `VariantCollection`?